### ISSUE TITLE
BRO - The Brothers' War Set Card Implementation Tracking

### ISSUE THREAD

**Issue by theelk801** [magefree/mage#9598]
This checklist is here to help manage the implementation of The Brothers' War. If a card is marked as being in progress then someone is working on it.

If you're new to implementing cards then you likely don't have permission to check things off. This is totally fine! We still appreciate your contributions so just leave a comment to let us know that you're working on it.

Don't worry about moving things from in progress to completed either, there's a script that handles that, and don't worry about fixing text issues as those are usually handled when the set is done.

[All Sets](https://github.com/magefree/mage/wiki/Set-implementation-list)

All cards are implemented

# Implemented Cards
<details><summary>Click to expand</summary>

- [x] Done -- [Aeronaut Cavalry](https://scryfall.com/search?q=!aeronautcavalry)
- [x] Done -- [Aeronaut's Wings](https://scryfall.com/search?q=!aeronautswings)
- [x] Done -- [Air Marshal](https://scryfall.com/search?q=!airmarshal)
- [x] Done -- [Airlift Chaplain](https://scryfall.com/search?q=!airliftchaplain)
- [x] Done -- [Alloy Animist](https://scryfall.com/search?q=!alloyanimist)
- [x] Done -- [Ambush Paratrooper](https://scryfall.com/search?q=!ambushparatrooper)
- [x] Done -- [Arbalest Engineers](https://scryfall.com/search?q=!arbalestengineers)
- [x] Done -- [Arcane Proxy](https://scryfall.com/search?q=!arcaneproxy)
- [x] Done -- [Argivian Avenger](https://scryfall.com/search?q=!argivianavenger)
- [x] Done -- [Argoth, Sanctum of Nature](https://scryfall.com/search?q=!argothsanctumofnature)
- [x] Done -- [Argothian Opportunist](https://scryfall.com/search?q=!argothianopportunist)
- [x] Done -- [Argothian Sprite](https://scryfall.com/search?q=!argothiansprite)
- [x] Done -- [Arms Race](https://scryfall.com/search?q=!armsrace)
- [x] Done -- [Artificer's Dragon](https://scryfall.com/search?q=!artificersdragon)
- [x] Done -- [Ashnod's Harvester](https://scryfall.com/search?q=!ashnodsharvester)
- [x] Done -- [Ashnod's Intervention](https://scryfall.com/search?q=!ashnodsintervention)
- [x] Done -- [Ashnod, Flesh Mechanist](https://scryfall.com/search?q=!ashnodfleshmechanist)
- [x] Done -- [Audacity](https://scryfall.com/search?q=!audacity)
- [x] Done -- [Autonomous Assembler](https://scryfall.com/search?q=!autonomousassembler)
- [x] Done -- [Awaken the Woods](https://scryfall.com/search?q=!awakenthewoods)
- [x] Done -- [Battery Bearer](https://scryfall.com/search?q=!batterybearer)
- [x] Done -- [Battlefield Butcher](https://scryfall.com/search?q=!battlefieldbutcher)
- [x] Done -- [Battlefield Forge](https://scryfall.com/search?q=!battlefieldforge)
- [x] Done -- [Bitter Reunion](https://scryfall.com/search?q=!bitterreunion)
- [x] Done -- [Bladecoil Serpent](https://scryfall.com/search?q=!bladecoilserpent)
- [x] Done -- [Blanchwood Armor](https://scryfall.com/search?q=!blanchwoodarmor)
- [x] Done -- [Blanchwood Prowler](https://scryfall.com/search?q=!blanchwoodprowler)
- [x] Done -- [Blast Zone](https://scryfall.com/search?q=!blastzone)
- [x] Done -- [Blitz Automaton](https://scryfall.com/search?q=!blitzautomaton)
- [x] Done -- [Boulderbranch Golem](https://scryfall.com/search?q=!boulderbranchgolem)
- [x] Done -- [Brotherhood's End](https://scryfall.com/search?q=!brotherhoodsend)
- [x] Done -- [Brushland](https://scryfall.com/search?q=!brushland)
- [x] Done -- [Burrowing Razormaw](https://scryfall.com/search?q=!burrowingrazormaw)
- [x] Done -- [Bushwhack](https://scryfall.com/search?q=!bushwhack)
- [x] Done -- [Calamity's Wake](https://scryfall.com/search?q=!calamityswake)
- [x] Done -- [Carrion Locust](https://scryfall.com/search?q=!carrionlocust)
- [x] Done -- [Citanul Stalwart](https://scryfall.com/search?q=!citanulstalwart)
- [x] Done -- [Cityscape Leveler](https://scryfall.com/search?q=!cityscapeleveler)
- [x] Done -- [Clay Champion](https://scryfall.com/search?q=!claychampion)
- [x] Done -- [Clay Revenant](https://scryfall.com/search?q=!clayrevenant)
- [x] Done -- [Coastal Bulwark](https://scryfall.com/search?q=!coastalbulwark)
- [x] Done -- [Combat Courier](https://scryfall.com/search?q=!combatcourier)
- [x] Done -- [Combat Thresher](https://scryfall.com/search?q=!combatthresher)
- [x] Done -- [Conscripted Infantry](https://scryfall.com/search?q=!conscriptedinfantry)
- [x] Done -- [Corrupt](https://scryfall.com/search?q=!corrupt)
- [x] Done -- [Cradle Clearcutter](https://scryfall.com/search?q=!cradleclearcutter)
- [x] Done -- [Curate](https://scryfall.com/search?q=!curate)
- [x] Done -- [Deadly Riposte](https://scryfall.com/search?q=!deadlyriposte)
- [x] Done -- [Deathbloom Ritualist](https://scryfall.com/search?q=!deathbloomritualist)
- [x] Done -- [Defabricate](https://scryfall.com/search?q=!defabricate)
- [x] Done -- [Demolition Field](https://scryfall.com/search?q=!demolitionfield)
- [x] Done -- [Depth Charge Colossus](https://scryfall.com/search?q=!depthchargecolossus)
- [x] Done -- [Desynchronize](https://scryfall.com/search?q=!desynchronize)
- [x] Done -- [Diabolic Intent](https://scryfall.com/search?q=!diabolicintent)
- [x] Done -- [Disciples of Gix](https://scryfall.com/search?q=!disciplesofgix)
- [x] Done -- [Disenchant](https://scryfall.com/search?q=!disenchant)
- [x] Done -- [Disfigure](https://scryfall.com/search?q=!disfigure)
- [x] Done -- [Draconic Destiny](https://scryfall.com/search?q=!draconicdestiny)
- [x] Done -- [Drafna, Founder of Lat-Nam](https://scryfall.com/search?q=!drafnafounderoflatnam)
- [x] Done -- [Dreams of Steel and Oil](https://scryfall.com/search?q=!dreamsofsteelandoil)
- [x] Done -- [Dredging Claw](https://scryfall.com/search?q=!dredgingclaw)
- [x] Done -- [Dwarven Forge-Chanter](https://scryfall.com/search?q=!dwarvenforgechanter)
- [x] Done -- [Emergency Weld](https://scryfall.com/search?q=!emergencyweld)
- [x] Done -- [Energy Refractor](https://scryfall.com/search?q=!energyrefractor)
- [x] Done -- [Epic Confrontation](https://scryfall.com/search?q=!epicconfrontation)
- [x] Done -- [Evangel of Synthesis](https://scryfall.com/search?q=!evangelofsynthesis)
- [x] Done -- [Evolving Wilds](https://scryfall.com/search?q=!evolvingwilds)
- [x] Done -- [Excavation Explosion](https://scryfall.com/search?q=!excavationexplosion)
- [x] Done -- [Fade from History](https://scryfall.com/search?q=!fadefromhistory)
- [x] Done -- [Fallaji Archaeologist](https://scryfall.com/search?q=!fallajiarchaeologist)
- [x] Done -- [Fallaji Chaindancer](https://scryfall.com/search?q=!fallajichaindancer)
- [x] Done -- [Fallaji Dragon Engine](https://scryfall.com/search?q=!fallajidragonengine)
- [x] Done -- [Fallaji Excavation](https://scryfall.com/search?q=!fallajiexcavation)
- [x] Done -- [Fallaji Vanguard](https://scryfall.com/search?q=!fallajivanguard)
- [x] Done -- [Fateful Handoff](https://scryfall.com/search?q=!fatefulhandoff)
- [x] Done -- [Fauna Shaman](https://scryfall.com/search?q=!faunashaman)
- [x] Done -- [Feldon, Ronom Excavator](https://scryfall.com/search?q=!feldonronomexcavator)
- [x] Done -- [Flow of Knowledge](https://scryfall.com/search?q=!flowofknowledge)
- [x] Done -- [Fog of War](https://scryfall.com/search?q=!fogofwar)
- [x] Done -- [Forging the Anchor](https://scryfall.com/search?q=!forgingtheanchor)
- [x] Done -- [Fortified Beachhead](https://scryfall.com/search?q=!fortifiedbeachhead)
- [x] Done -- [Gaea's Courser](https://scryfall.com/search?q=!gaeascourser)
- [x] Done -- [Gaea's Gift](https://scryfall.com/search?q=!gaeasgift)
- [x] Done -- [Geology Enthusiast](https://scryfall.com/search?q=!geologyenthusiast)
- [x] Done -- [Giant Cindermaw](https://scryfall.com/search?q=!giantcindermaw)
- [x] Done -- [Giant Growth](https://scryfall.com/search?q=!giantgrowth)
- [x] Done -- [Gix's Caress](https://scryfall.com/search?q=!gixscaress)
- [x] Done -- [Gix's Command](https://scryfall.com/search?q=!gixscommand)
- [x] Done -- [Gix, Yawgmoth Praetor](https://scryfall.com/search?q=!gixyawgmothpraetor)
- [x] Done -- [Gixian Infiltrator](https://scryfall.com/search?q=!gixianinfiltrator)
- [x] Done -- [Gixian Puppeteer](https://scryfall.com/search?q=!gixianpuppeteer)
- [x] Done -- [Gixian Skullflayer](https://scryfall.com/search?q=!gixianskullflayer)
- [x] Done -- [Gnarlroot Pallbearer](https://scryfall.com/search?q=!gnarlrootpallbearer)
- [x] Done -- [Gnawing Vermin](https://scryfall.com/search?q=!gnawingvermin)
- [x] Done -- [Go for the Throat](https://scryfall.com/search?q=!goforthethroat)
- [x] Done -- [Goblin Blast-Runner](https://scryfall.com/search?q=!goblinblastrunner)
- [x] Done -- [Goblin Firebomb](https://scryfall.com/search?q=!goblinfirebomb)
- [x] Done -- [Goring Warplow](https://scryfall.com/search?q=!goringwarplow)
- [x] Done -- [Great Desert Prospector](https://scryfall.com/search?q=!greatdesertprospector)
- [x] Done -- [Gruesome Realization](https://scryfall.com/search?q=!gruesomerealization)
- [x] Done -- [Gurgling Anointer](https://scryfall.com/search?q=!gurglinganointer)
- [x] Done -- [Gwenna, Eyes of Gaea](https://scryfall.com/search?q=!gwennaeyesofgaea)
- [x] Done -- [Hajar, Loyal Bodyguard](https://scryfall.com/search?q=!hajarloyalbodyguard)
- [x] Done -- [Hall of Tagsin](https://scryfall.com/search?q=!halloftagsin)
- [x] Done -- [Harbin, Vanguard Aviator](https://scryfall.com/search?q=!harbinvanguardaviator)
- [x] Done -- [Haywire Mite](https://scryfall.com/search?q=!haywiremite)
- [x] Done -- [Heavyweight Demolisher](https://scryfall.com/search?q=!heavyweightdemolisher)
- [x] Done -- [Hero of the Dunes](https://scryfall.com/search?q=!heroofthedunes)
- [x] Done -- [Hoarding Recluse](https://scryfall.com/search?q=!hoardingrecluse)
- [x] Done -- [Horned Stoneseeker](https://scryfall.com/search?q=!hornedstoneseeker)
- [x] Done -- [Hostile Negotiations](https://scryfall.com/search?q=!hostilenegotiations)
- [x] Done -- [Hulking Metamorph](https://scryfall.com/search?q=!hulkingmetamorph)
- [x] Done -- [Hurkyl's Final Meditation](https://scryfall.com/search?q=!hurkylsfinalmeditation)
- [x] Done -- [Hurkyl, Master Wizard](https://scryfall.com/search?q=!hurkylmasterwizard)
- [x] Done -- [In the Trenches](https://scryfall.com/search?q=!inthetrenches)
- [x] Done -- [Involuntary Cooldown](https://scryfall.com/search?q=!involuntarycooldown)
- [x] Done -- [Iron-Craw Crusher](https://scryfall.com/search?q=!ironcrawcrusher)
- [x] Done -- [Junkyard Genius](https://scryfall.com/search?q=!junkyardgenius)
- [x] Done -- [Kayla's Command](https://scryfall.com/search?q=!kaylascommand)
- [x] Done -- [Kayla's Reconstruction](https://scryfall.com/search?q=!kaylasreconstruction)
- [x] Done -- [Keeper of the Cadence](https://scryfall.com/search?q=!keeperofthecadence)
- [x] Done -- [Kill-Zone Acrobat](https://scryfall.com/search?q=!killzoneacrobat)
- [x] Done -- [Koilos Roc](https://scryfall.com/search?q=!koilosroc)
- [x] Done -- [Lat-Nam Adept](https://scryfall.com/search?q=!latnamadept)
- [x] Done -- [Lay Down Arms](https://scryfall.com/search?q=!laydownarms)
- [x] Done -- [Legions to Ashes](https://scryfall.com/search?q=!legionstoashes)
- [x] Done -- [Levitating Statue](https://scryfall.com/search?q=!levitatingstatue)
- [x] Done -- [Liberator, Urza's Battlethopter](https://scryfall.com/search?q=!liberatorurzasbattlethopter)
- [x] Done -- [Llanowar Wastes](https://scryfall.com/search?q=!llanowarwastes)
- [x] Done -- [Loran of the Third Path](https://scryfall.com/search?q=!loranofthethirdpath)
- [x] Done -- [Loran's Escape](https://scryfall.com/search?q=!loransescape)
- [x] Done -- [Loran, Disciple of History](https://scryfall.com/search?q=!lorandiscipleofhistory)
- [x] Done -- [Machine Over Matter](https://scryfall.com/search?q=!machineovermatter)
- [x] Done -- [Mask of the Jadecrafter](https://scryfall.com/search?q=!maskofthejadecrafter)
- [x] Done -- [Mass Production](https://scryfall.com/search?q=!massproduction)
- [x] Done -- [Mechanized Warfare](https://scryfall.com/search?q=!mechanizedwarfare)
- [x] Done -- [Meticulous Excavation](https://scryfall.com/search?q=!meticulousexcavation)
- [x] Done -- [Mightstone's Animation](https://scryfall.com/search?q=!mightstonesanimation)
- [x] Done -- [Military Discipline](https://scryfall.com/search?q=!militarydiscipline)
- [x] Done -- [Mine Worker](https://scryfall.com/search?q=!mineworker)
- [x] Done -- [Misery's Shadow](https://scryfall.com/search?q=!miserysshadow)
- [x] Done -- [Mishra's Command](https://scryfall.com/search?q=!mishrascommand)
- [x] Done -- [Mishra's Domination](https://scryfall.com/search?q=!mishrasdomination)
- [x] Done -- [Mishra's Foundry](https://scryfall.com/search?q=!mishrasfoundry)
- [x] Done -- [Mishra's Juggernaut](https://scryfall.com/search?q=!mishrasjuggernaut)
- [x] Done -- [Mishra's Onslaught](https://scryfall.com/search?q=!mishrasonslaught)
- [x] Done -- [Mishra's Research Desk](https://scryfall.com/search?q=!mishrasresearchdesk)
- [x] Done -- [Mishra, Claimed by Gix](https://scryfall.com/search?q=!mishraclaimedbygix)
- [x] Done -- [Mishra, Excavation Prodigy](https://scryfall.com/search?q=!mishraexcavationprodigy)
- [x] Done -- [Mishra, Lost to Phyrexia](https://scryfall.com/search?q=!mishralosttophyrexia)
- [x] Done -- [Mishra, Tamer of Mak Fawa](https://scryfall.com/search?q=!mishratamerofmakfawa)
- [x] Done -- [Moment of Defiance](https://scryfall.com/search?q=!momentofdefiance)
- [x] Done -- [Monastery Swiftspear](https://scryfall.com/search?q=!monasteryswiftspear)
- [x] Done -- [Myrel, Shield of Argive](https://scryfall.com/search?q=!myrelshieldofargive)
- [x] Done -- [No One Left Behind](https://scryfall.com/search?q=!nooneleftbehind)
- [x] Done -- [Obliterating Bolt](https://scryfall.com/search?q=!obliteratingbolt)
- [x] Done -- [Obstinate Baloth](https://scryfall.com/search?q=!obstinatebaloth)
- [x] Done -- [One with the Multiverse](https://scryfall.com/search?q=!onewiththemultiverse)
- [x] Done -- [Over the Top](https://scryfall.com/search?q=!overthetop)
- [x] Done -- [Overwhelming Remorse](https://scryfall.com/search?q=!overwhelmingremorse)
- [x] Done -- [Painful Quandary](https://scryfall.com/search?q=!painfulquandary)
- [x] Done -- [Penregon Strongbull](https://scryfall.com/search?q=!penregonstrongbull)
- [x] Done -- [Perennial Behemoth](https://scryfall.com/search?q=!perennialbehemoth)
- [x] Done -- [Perimeter Patrol](https://scryfall.com/search?q=!perimeterpatrol)
- [x] Done -- [Phalanx Vanguard](https://scryfall.com/search?q=!phalanxvanguard)
- [x] Done -- [Phyrexian Dragon Engine](https://scryfall.com/search?q=!phyrexiandragonengine)
- [x] Done -- [Phyrexian Fleshgorger](https://scryfall.com/search?q=!phyrexianfleshgorger)
- [x] Done -- [Platoon Dispenser](https://scryfall.com/search?q=!platoondispenser)
- [x] Done -- [Portal to Phyrexia](https://scryfall.com/search?q=!portaltophyrexia)
- [x] Done -- [Power Plant Worker](https://scryfall.com/search?q=!powerplantworker)
- [x] Done -- [Powerstone Engineer](https://scryfall.com/search?q=!powerstoneengineer)
- [x] Done -- [Powerstone Fracture](https://scryfall.com/search?q=!powerstonefracture)
- [x] Done -- [Prison Sentence](https://scryfall.com/search?q=!prisonsentence)
- [x] Done -- [Pyrrhic Blast](https://scryfall.com/search?q=!pyrrhicblast)
- [x] Done -- [Queen Kayla bin-Kroog](https://scryfall.com/search?q=!queenkaylabinkroog)
- [x] Done -- [Ravenous Gigamole](https://scryfall.com/search?q=!ravenousgigamole)
- [x] Done -- [Raze to the Ground](https://scryfall.com/search?q=!razetotheground)
- [x] Done -- [Razorlash Transmogrant](https://scryfall.com/search?q=!razorlashtransmogrant)
- [x] Done -- [Recommission](https://scryfall.com/search?q=!recommission)
- [x] Done -- [Reconstructed Thopter](https://scryfall.com/search?q=!reconstructedthopter)
- [x] Done -- [Recruitment Officer](https://scryfall.com/search?q=!recruitmentofficer)
- [x] Done -- [Repair and Recharge](https://scryfall.com/search?q=!repairandrecharge)
- [x] Done -- [Rescue Retriever](https://scryfall.com/search?q=!rescueretriever)
- [x] Done -- [Retrieval Agent](https://scryfall.com/search?q=!retrievalagent)
- [x] Done -- [Roc Hunter](https://scryfall.com/search?q=!rochunter)
- [x] Done -- [Rootwire Amalgam](https://scryfall.com/search?q=!rootwireamalgam)
- [x] Done -- [Rust Goliath](https://scryfall.com/search?q=!rustgoliath)
- [x] Done -- [Saheeli, Filigree Master](https://scryfall.com/search?q=!saheelifiligreemaster)
- [x] Done -- [Sardian Cliffstomper](https://scryfall.com/search?q=!sardiancliffstomper)
- [x] Done -- [Sarinth Greatwurm](https://scryfall.com/search?q=!sarinthgreatwurm)
- [x] Done -- [Sarinth Steelseeker](https://scryfall.com/search?q=!sarinthsteelseeker)
- [x] Done -- [Scatter Ray](https://scryfall.com/search?q=!scatterray)
- [x] Done -- [Scrapwork Cohort](https://scryfall.com/search?q=!scrapworkcohort)
- [x] Done -- [Scrapwork Mutt](https://scryfall.com/search?q=!scrapworkmutt)
- [x] Done -- [Scrapwork Rager](https://scryfall.com/search?q=!scrapworkrager)
- [x] Done -- [Shoot Down](https://scryfall.com/search?q=!shootdown)
- [x] Done -- [Sibling Rivalry](https://scryfall.com/search?q=!siblingrivalry)
- [x] Done -- [Siege Veteran](https://scryfall.com/search?q=!siegeveteran)
- [x] Done -- [Simian Simulacrum](https://scryfall.com/search?q=!simiansimulacrum)
- [x] Done -- [Skitterbeam Battalion](https://scryfall.com/search?q=!skitterbeambattalion)
- [x] Done -- [Skyfisher Spider](https://scryfall.com/search?q=!skyfisherspider)
- [x] Done -- [Skystrike Officer](https://scryfall.com/search?q=!skystrikeofficer)
- [x] Done -- [Slagstone Refinery](https://scryfall.com/search?q=!slagstonerefinery)
- [x] Done -- [Soul Partition](https://scryfall.com/search?q=!soulpartition)
- [x] Done -- [Spectrum Sentinel](https://scryfall.com/search?q=!spectrumsentinel)
- [x] Done -- [Splitting the Powerstone](https://scryfall.com/search?q=!splittingthepowerstone)
- [x] Done -- [Spotter Thopter](https://scryfall.com/search?q=!spotterthopter)
- [x] Done -- [Static Net](https://scryfall.com/search?q=!staticnet)
- [x] Done -- [Steel Exemplar](https://scryfall.com/search?q=!steelexemplar)
- [x] Done -- [Steel Seraph](https://scryfall.com/search?q=!steelseraph)
- [x] Done -- [Stern Lesson](https://scryfall.com/search?q=!sternlesson)
- [x] Done -- [Stone Retrieval Unit](https://scryfall.com/search?q=!stoneretrievalunit)
- [x] Done -- [Su-Chi Cave Guard](https://scryfall.com/search?q=!suchicaveguard)
- [x] Done -- [Supply Drop](https://scryfall.com/search?q=!supplydrop)
- [x] Done -- [Surge Engine](https://scryfall.com/search?q=!surgeengine)
- [x] Done -- [Survivor of Korlis](https://scryfall.com/search?q=!survivorofkorlis)
- [x] Done -- [Swiftgear Drake](https://scryfall.com/search?q=!swiftgeardrake)
- [x] Done -- [Symmetry Matrix](https://scryfall.com/search?q=!symmetrymatrix)
- [x] Done -- [Take Flight](https://scryfall.com/search?q=!takeflight)
- [x] Done -- [Tawnos's Tinkering](https://scryfall.com/search?q=!tawnosstinkering)
- [x] Done -- [Tawnos, the Toymaker](https://scryfall.com/search?q=!tawnosthetoymaker)
- [x] Done -- [Teething Wurmlet](https://scryfall.com/search?q=!teethingwurmlet)
- [x] Done -- [Teferi, Temporal Pilgrim](https://scryfall.com/search?q=!teferitemporalpilgrim)
- [x] Done -- [Terisian Mindbreaker](https://scryfall.com/search?q=!terisianmindbreaker)
- [x] Done -- [Terror Ballista](https://scryfall.com/search?q=!terrorballista)
- [x] Done -- [The Fall of Kroog](https://scryfall.com/search?q=!thefallofkroog)
- [x] Done -- [The Mightstone and Weakstone](https://scryfall.com/search?q=!themightstoneandweakstone)
- [x] Done -- [The Stasis Coffin](https://scryfall.com/search?q=!thestasiscoffin)
- [x] Done -- [The Stone Brain](https://scryfall.com/search?q=!thestonebrain)
- [x] Done -- [The Temporal Anchor](https://scryfall.com/search?q=!thetemporalanchor)
- [x] Done -- [Third Path Iconoclast](https://scryfall.com/search?q=!thirdpathiconoclast)
- [x] Done -- [Third Path Savant](https://scryfall.com/search?q=!thirdpathsavant)
- [x] Done -- [Thopter Architect](https://scryfall.com/search?q=!thopterarchitect)
- [x] Done -- [Thopter Mechanic](https://scryfall.com/search?q=!thoptermechanic)
- [x] Done -- [Thran Power Suit](https://scryfall.com/search?q=!thranpowersuit)
- [x] Done -- [Thran Spider](https://scryfall.com/search?q=!thranspider)
- [x] Done -- [Thran Vigil](https://scryfall.com/search?q=!thranvigil)
- [x] Done -- [Thraxodemon](https://scryfall.com/search?q=!thraxodemon)
- [x] Done -- [Titania's Command](https://scryfall.com/search?q=!titaniascommand)
- [x] Done -- [Titania, Gaea Incarnate](https://scryfall.com/search?q=!titaniagaeaincarnate)
- [x] Done -- [Titania, Voice of Gaea](https://scryfall.com/search?q=!titaniavoiceofgaea)
- [x] Done -- [Tocasia's Dig Site](https://scryfall.com/search?q=!tocasiasdigsite)
- [x] Done -- [Tocasia's Onulet](https://scryfall.com/search?q=!tocasiasonulet)
- [x] Done -- [Tocasia's Welcome](https://scryfall.com/search?q=!tocasiaswelcome)
- [x] Done -- [Tocasia, Dig Site Mentor](https://scryfall.com/search?q=!tocasiadigsitementor)
- [x] Done -- [Tomakul Honor Guard](https://scryfall.com/search?q=!tomakulhonorguard)
- [x] Done -- [Tomakul Scrapsmith](https://scryfall.com/search?q=!tomakulscrapsmith)
- [x] Done -- [Tower Worker](https://scryfall.com/search?q=!towerworker)
- [x] Done -- [Transmogrant Altar](https://scryfall.com/search?q=!transmograntaltar)
- [x] Done -- [Transmogrant's Crown](https://scryfall.com/search?q=!transmograntscrown)
- [x] Done -- [Trench Stalker](https://scryfall.com/search?q=!trenchstalker)
- [x] Done -- [Tyrant of Kher Ridges](https://scryfall.com/search?q=!tyrantofkherridges)
- [x] Done -- [Underground River](https://scryfall.com/search?q=!undergroundriver)
- [x] Done -- [Union of the Third Path](https://scryfall.com/search?q=!unionofthethirdpath)
- [x] Done -- [Unleash Shell](https://scryfall.com/search?q=!unleashshell)
- [x] Done -- [Urza's Command](https://scryfall.com/search?q=!urzascommand)
- [x] Done -- [Urza's Rebuff](https://scryfall.com/search?q=!urzasrebuff)
- [x] Done -- [Urza's Sylex](https://scryfall.com/search?q=!urzassylex)
- [x] Done -- [Urza, Lord Protector](https://scryfall.com/search?q=!urzalordprotector)
- [x] Done -- [Urza, Planeswalker](https://scryfall.com/search?q=!urzaplaneswalker)
- [x] Done -- [Urza, Powerstone Prodigy](https://scryfall.com/search?q=!urzapowerstoneprodigy)
- [x] Done -- [Urza, Prince of Kroog](https://scryfall.com/search?q=!urzaprinceofkroog)
- [x] Done -- [Veteran's Powerblade](https://scryfall.com/search?q=!veteranspowerblade)
- [x] Done -- [Visions of Phyrexia](https://scryfall.com/search?q=!visionsofphyrexia)
- [x] Done -- [Warlord's Elite](https://scryfall.com/search?q=!warlordselite)
- [x] Done -- [Wasteful Harvest](https://scryfall.com/search?q=!wastefulharvest)
- [x] Done -- [Weakstone's Subjugation](https://scryfall.com/search?q=!weakstonessubjugation)
- [x] Done -- [Whirling Strike](https://scryfall.com/search?q=!whirlingstrike)
- [x] Done -- [Wing Commando](https://scryfall.com/search?q=!wingcommando)
- [x] Done -- [Woodcaller Automaton](https://scryfall.com/search?q=!woodcallerautomaton)
- [x] Done -- [Yotian Dissident](https://scryfall.com/search?q=!yotiandissident)
- [x] Done -- [Yotian Frontliner](https://scryfall.com/search?q=!yotianfrontliner)
- [x] Done -- [Yotian Medic](https://scryfall.com/search?q=!yotianmedic)
- [x] Done -- [Yotian Tactician](https://scryfall.com/search?q=!yotiantactician)
- [x] Done -- [Zephyr Sentinel](https://scryfall.com/search?q=!zephyrsentinel)
</details>

**Comment by theelk801** [comment]
Additional tasks: 

None yet

**Comment by lunelis** [comment]
Hello, I started working on cards without seeing the thread. As a newcomer to the project, I was looking for easy cards to implement so I started with Fortified Beachhead. I see that it is already in progress, should I drop it ? I was also interested in The Fall of Kroog, Recommission or Rescue Retriever.
If you think one is easier than the other let me know and I'll work on it !

**Comment by theelk801** [comment]
unfortunately were at the point where all the easiest cards are done, but of the ones you listed I would go with Rescue Retriever

**Comment by lunelis** [comment]
No problem, I'll work on Rescue Retriever then.

**Comment by chaoticmachinery** [comment]
Phyrexian Fleshgorger does not show up in the deck editor.  It shows up as done in the implemented card list on github.

**Comment by Grath** [comment]
> Phyrexian Fleshgorger does not show up in the deck editor. It shows up as done in the implemented card list on github.

The implementation of Prototype needs to be merged before any cards with Prototype can show up.

**Comment by MorihiroS** [comment]
The Stone Brain is already done, am I right? When is it planned to add it to XMage Beta?

**Comment by JayDi85** [comment]
@MorihiroS not yet, but you can follow for it in #9727

**Comment by MorihiroS** [comment]
@JayDi85 , I saw, that #9765 , #9763 , #9758 have "All checks have passed" and "This branch has no conflicts with the base branch". Only #9727 have "This branch has conflicts that must be resolved", but the problem of this card is similar to the problem of another card, Unmoored Ego, which is already(!) in the game. And The Stone Brain is important for some decks, even in older formats. From what I understand, it works fine, it just shares a bug with Unmoored Ego that doesn't interfere with the game too much. Is it possible to add it as it's already done?

**Comment by JayDi85** [comment]
@MorihiroS I think @theelk801 can help with new cards merge.

**Comment by theelk801** [comment]
yeah I'm still getting caught up on PRs

**Comment by theelk801** [comment]
Leaving this open until prototype is implemented

**Comment by xenohedron** [comment]
Prototype implemented in #11249 , collation implemented in #10364
